app/controllers/custom_css_controller.rb

```diff
@@ -0,0 +1,23 @@
+class CustomCssController < ApplicationController
+  def show
+    render text: generate_custom_css(current_site), content_type: 'text/css'
+  end
+
+  private
+
+  def generate_custom_css(site)
+    path = Rails.root.join("tmp/custom_css/#{site.id}")
+
+    FileUtils.mkdir_p path
+
+    css_path = path.join('application.scss')
+
+    File.write css_path, <<~SCSS
```

Comment:
Yes! This seems a perfect idea.
I totally agreed :100: 

However I have some refactoring, and I need to copy directory.
So I use `Dir.mktmpdir` instead :sunglasses: